850.split-ambiguities/y.yaml

```diff
@@ -6,6 +6,10 @@
 - { name: yali, wwwpart: pisi, setname: yali-installer }
 - { name: yali, addflag: unclassified }
 
+- { name: yass, wwwpart: kokkinizita, setname: yet-another-scrolling-scope }
+- { name: yass, wwwpart: bioinfo, setname: yet-another-similarity-searcher }
+- { name: yass, wwwpart: yass-along, setname: yass-karaoke }
```

Comment:
Every block of split rules must end with `addflag: unclassified` rule.

Reply:
Ah missed that